src/olympia/shelves/tests/test_serializers.py

```diff
@@ -295,6 +295,12 @@ def test_tags_shelf(self):
         assert data['addons'][0]['name'] == {'en-US': 'test addon test04'}
         assert data['addons'][1]['name'] == {'en-US': 'test addon test01'}
 
+    def test_tag_token_substitution(self):
+        self.tag_shelf.update(title='Title for {tag}', footer_text='footer with {tag}')
+        data = self.serialize(self.tag_shelf)
+        assert data['title'] == {'en-US': 'Title for foo'}
+        assert data['footer']['text'] == {'en-US': 'footer with foo'}
```

Comment:
Could we have a test case with `{tag}` in a footer text/title for a shelf that isn't for "random tag"?